doc/tutorials/contributing-clion.md

```diff
@@ -0,0 +1,183 @@
+# Introduction
+
+In this tutorial, we will go through the steps necessary to contribute to
+Elektra to make it easier for you to get started. We will use a Unix based OS
+like Linux and CLion for development, but depending on where you want to
+contribute code, other IDEs will also be sufficient. Before you start, please
+read [this](/.github/CONTRIBUTING.md) to get familiar with the process of
+contributing to Elektra.
+
+## Prerequisites
+
+- Git ([link](https://git-scm.com/download)) or similar software
+- CLion ([link](https://www.jetbrains.com/clion/)) or similar software
+- GitHub-account ([link](https://github.com/))
+
+## Forking the repository
+
+Libelektra is hosted on GitHub. You can find its repository here:
+
+- https://github.com/ElektraInitiative/libelektra
+
+To be able to make pull requests, you need a copy of this repository inside
+your GitHub-account. You can find a tutorial about how to do this
+[here](https://help.github.com/en/articles/fork-a-repo) (remember to sign
+into your account first). After this, you should see this copy in the list
+of your own repositories with a hint of it's origin.
+
+## Getting the source-code
+
+To develop for libelektra, we now have to "download" your copy of it's
+original repository. In git this process is called "cloning". Open a
+terminal and navigate to the folder you want to save the source code into
+and type:
+
+```sh
+git clone https://github.com/ElektraInitiative/libelektra.git
+```
+
+With the project now locally available we can start developing.
+
+## Setting up the project
+
+It is important to start CLion as root-user, since some functionalities of
+_kdb_ require superuser privileges to work as expected. Once you see the
+main menu, click "Open" and select the CMakeLists.txt-file inside the
+project's root directory. This will import the project accordingly and
+populate you run-configuration with some predefined values. In rare
+occurrences this won't happen. If that is the case for you, simply restart
+CLion using:
+
+_File_ --> _Invalidate Caches / Restart..._ --> _Invalidate and Restart_
+
+Now after all processes of CLion have finished, the project should be set up
+and the run-configuration should be populated with entries.
+
+## Development
+
+Usually the folders you have to work in to add functionality or
+documentation are as follows:
+
+- doc<br/>
```

Comment:
Don't use `<br />`, a newline should also do the job. To continue the enumeration after the line break, indent the text.

```markdown
- doc
  This folder contains mainly all documentation of the project, including
```

Reply:
> a newline should also do the job

Not every markdown renderer accepts the newline as new line, `<br>` should be fine here.